pkgs/development/tools/rust/cargo-insta/default.nix

```diff
@@ -5,18 +5,20 @@
 
 rustPlatform.buildRustPackage rec {
   pname = "cargo-insta";
-  version = "1.33.0";
+  # use stable release once 1.40 lands
+  version = "1.39-unstable-2024-08-04";
 
   src = fetchFromGitHub {
     owner = "mitsuhiko";
     repo = "insta";
-    rev = "refs/tags/${version}";
-    hash = "sha256-w/dxIQ7KRrn86PwiE/g5L9Gn8KszPF9u/zlwE/FYDu4=";
+    # rev = "refs/tags/${version}";
```

Comment:
Leaving this in comment has no use for either Nix or human. I'd remove it. One can `git blame` to get its history anyway.